### PR TITLE
Score of SPOOF_REPLYTO is 3

### PR DESCRIPTION
**Pull request by stephdl**
force the weight of SPOOF_REPLYTO to 3

### CODE REVIEW

File: filter/etc/rspamd/scores.d/nethserver_group.conf [hunk 1]
```diff
@@ -1,11 +1,14 @@
 #
 # This file will be overwritten by the next rpm update
 # use /etc/rspamd/local.d/nethserver_group.conf' - to override the defaults
-#
+# 
 
 symbols = {
     "IP_REPUTATION" {
-        score = 0.0;
+        weight = 0.0;
```
Comment (gsanchietti) [inline filter/etc/rspamd/scores.d/nethserver_group.conf:8]: Why this change?

File: filter/etc/rspamd/scores.d/nethserver_group.conf [hunk 2]
```diff
@@ -1,11 +1,14 @@
 #
 # This file will be overwritten by the next rpm update
 # use /etc/rspamd/local.d/nethserver_group.conf' - to override the defaults
-#
+#
```
Comment (gsanchietti) [inline filter/etc/rspamd/scores.d/nethserver_group.conf:4]: Please, try to avoid useless modifications ;)

File: filter/etc/rspamd/scores.d/nethserver_group.conf (hunk 1 above)
Comment (stephdl) [reply filter/etc/rspamd/scores.d/nethserver_group.conf:8]: https://rspamd.com/doc/configuration/metrics.html#symbols

do not know, vlad suck or delabrusse suck :)

It is workable with score, but doc now states weight

File: filter/etc/rspamd/scores.d/nethserver_group.conf (hunk 2 above)
Comment (stephdl) [reply filter/etc/rspamd/scores.d/nethserver_group.conf]: done
